controllers/pvc-mutator/README.md

```diff
@@ -0,0 +1,62 @@
+# PVC Mutator Admission Controller
+
+The PVC Mutator is a mutating admission controller that modifies
+PersistentVolumeClaims during the create process.
+
+## Mutators
+
+The PVC Mutator can run multiple mutation functions, each performing a different
+task:
+
+- TODO(croomes): List of mutators should be added here.
+
+## Webhook server
+
+The admission controller runs as a webhook within api-manager.
+The webhook server uses a self-signed certificate.
+
+Certificates are rotated automatically and stored in a secret.  Multiple
+instances of the api-manager share the same certificate and will check for
+updates periodically, configured with the `-webhook-cert-refresh-interval` flag.
```

Comment:
This seems repetitive. We have similar paragraphs in pod mutator too. It'll increase docs maintenance work.

Reply:
The following paragraphs seem to be the same as in the pod mutator. Maybe we should have a common docs for all the mutators.

Reply:
Agreed.  I've moved the webhook docs into the main `README`.  It should be a lot less repetitive now.